pages/news-and-views/page/_num/index.vue

```diff
@@ -2,7 +2,7 @@
 	<article>
 		<h1>{{ title }}</h1>
 		<NewsGrid :postList="pagePostList[parseInt($route.params.num) - 1]" />
-		<Pagination v-if="pageCount > 1" :pageLinks="pageLinks" :currentPageNum="parseInt($route.params.num)" />
+		<Pagination v-if="pageCount > 1" :pageLinks="pageLinks" :currentPageNum="parseInt($route.params.num)" class="test" />
```

Comment:
This `class="test"` seems to be debugging code that was added and then not removed. Is it needed?

Reply:
Correct, thanks for the catch